test/lib/code_corps/validators/slug_validator_test.exs

```diff
@@ -88,6 +88,21 @@ defmodule CodeCorps.Validators.SlugValidatorTest do
     refute changeset.valid?
   end
 
+  test "reserves all api routes" do
+    CodeCorpsWeb.Router.__routes__
+    |> Enum.map(&Map.get(&1, :path))
+    |> Enum.map(&String.split(&1, "/"))
```

Comment:
This means that a route like `/foo/bar/baz` will reserve `foo`, `bar`and `baz`, not just `foo`.

If that's too strict, I can loosen it by getting the first element of each list here.